javascript/javascript3/week3/shopping-cart.js

```diff
@@ -0,0 +1,189 @@
+const userName = document.getElementById("name")
+
+class Product {
+    constructor(name, price) {
+        this.name = name;
+        this.price = price;
+    }
+    convertToCurrency(currency) {
+        switch (currency) {
+            case "USD":
+                this.price *= 0.16
+                break;
+            case "EUR":
+                this.price *= 0.13
+                break;
+            case "AUD":
+                this.price *= 0.2
+                break;
+            case "BGN":
+                this.price *= 0.26
+                break;
+            case "BRL":
+                this.price *= 0.9
+                break;
+            case "CAD":
+                this.price *= 0.19
+                break;
+            case "CHF":
+                this.price *= 0.14
+                break;
+            case "CNY":
+                this.price *= 1.03
+                break;
+            case "CZK":
+                this.price *= 3.5
+                break;
+            case "GBP":
+                this.price *= 0.11
+                break;
+            case "HKD":
+                this.price *= 1.23
+                break;
+            case "HRK":
+                this.price *= 1.01
+                break;
+            case "HUF":
+                this.price *= 48.89
+                break;
+            case "IDR":
+                this.price *= 2284.35
+                break;
+            case "ILS":
+                this.price *= 0.52
+                break;
+            case "INR":
+                this.price *= 11.5
+                break;
+            case "ISK":
+                this.price *= 20.14
+                break;
+            case "JPY":
+                this.price *= 17.39
+                break;
+            case "KRW":
+                this.price *= 179.15
+                break;
+            case "MXN":
+                this.price *= 3.26
+                break;
+            case "MYR":
+                this.price *= 0.65
+                break;
+            case "NOK":
+                this.price *= 1.36
+                break;
+            case "NZD":
+                this.price *= 0.22
+                break;
+            case "PHP":
+                this.price *= 7.68
+                break;
+            case "PLN":
+                this.price *= 0.62
+                break;
+            case "RON":
+                this.price *= 0.65
+                break;
+            case "RUB":
+                this.price *= 12.03
+                break;
+            case "SEK":
+                this.price *= 1.37
+                break;
+            case "SGD":
+                this.price *= 0.21
+                break;
+            case "THB":
+                this.price *= 4.92
+                break;
+            case "TRY":
+                this.price *= 1.27
+                break;
+            case "ZAR":
+                this.price *= 2.37
+        }
+        return this.price.toFixed(2);
+    }
+}
```

Comment:
lot of work with currency exchange! :) 
did you look on some APIs you can use for accurate exchange rates?

Reply:
Yes, but got some errors with it.